### DISCUSSION TITLE
Process Revit Files on Bentley's Cloud

### DISCUSSION THREAD

**Discussion by youngbrett** [iTwin/itwinjs-core#3875] [General]
Hello --

What services do I need from Bentley to be able to process Revit files into iTwins using Bentley's cloud?  My (very selfish) use case  is I am on a Mac and one of the few uses I have for the PC is to process Revit files using the iTwin Synchronizer.  I'd like to be able to push a Revit file to Bentley's cloud and be able to process it from there in the near term.  I can also imagine use cases where processing files on the cloud has advantages.  

Thanks in advance,
Brett.

**Comment by jffmarker** [comment]
Are you asking from a programmatic standpoint? Try taking a look at https://developer.bentley.com/apis/synchronization/. There are also some tutorials at https://developer.bentley.com/apis/synchronization/tutorials/.
@timlawrence-bentley @Laisvune
